src/dftbp/api/mm/dftbplus.h

```diff
@@ -278,6 +288,26 @@ void dftbp_get_energy(DftbPlus *instance, double *mermin_energy);
 void dftbp_get_gradients(DftbPlus *instance, double *gradients);
 
 
+/**
+ * Queries the number of basis functions for each atom in current geometry.
+ *
+ * \param[inout] instance Handler of the DFTB+ instance.
+ *
+ * \param[out] nOrbitals number of orbitals on each atom
+ */
+void dftbp_get_nr_basisfns(DftbPlus *instance, int *nOrbitals);
```

Comment:
```suggestion
void dftbp_get_nr_orbitals(DftbPlus *instance, int *nOrbitals);
```

Reply:
Probably _get_nr_basisfns is more accurate in the case of non-minimal basis (as they don't neccessarily correspond to atomic orbitals. and also nr_orbitals could be mistaned with total matrix order), but I take the point in terms of matching internal variable names.